scripts/deploy/create_github_release.rb

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env ruby
+
+require 'octokit'
+require 'subprocess'
+
+require_relative 'common'
+require_relative 'gnupg_utils'
+
+@release_url = nil
+
+def create_github_release
+    build_example_release_apk
+    tag_release
+
+    begin
+        release_response = octokit_client.create_release(
+          "stripe/stripe-android",
+          "v#{@version}",
+          name: "stripe-android v#{@version}",
+          body: release_description,
+          draft: @is_dry_run
+        )
+
+        @release_url = release_response.url
+        upload_example_apk_to_release(@release_url)
+
+        rputs "Created new release"
+        open_url(release_response.html_url)
+    rescue
+        delete_release_tag
+    end
+end
+
+def delete_github_release
```

Comment:
Should we try/catch these too?

Reply:
If creation fails due to network failure or permissions, it'll likely fail this too, then what happens?

Reply:
These don't throw exceptions, so we don't need to try/catch them. If they fail, they'll output an error in the terminal. I think that's the right thing to do here, because it ensures that we _try_ to clean up everything and are notified if something doesn't work.

Reply:
I added a new error message for deleting the release, because that used to silently fail. 

Deleting the release tag already outputs an error